README.md

```diff
@@ -11,61 +11,118 @@
+
+This starter application will show how easy it is to deploy an application both locally and in a remote environment.
+
+## Pre-reqs
+* Install [Docker](https://docs.docker.com/get-docker/) and make sure it's running
+* Install the [Architect CLI](https://github.com/architect-team/architect-cli)
+* [Sign up for a free Architect account](https://cloud.architect.io/signup)
+
+## Clone the repo
+You can clone this repo yourself or use the `architect init` command to use this project.
+
+### Use `architect init`
+You can clone this repo locally using the `architect init` command. A drop-down list of Starter Projects is
+displayed for you to select from.
+
+```bash
+$ architect init
+? What is the name of your project? my-starter-project
+? Please select a framework/language for your project NodeJS
+
+######################################
+##### Let's set up your project! #####
+######################################
+
+Creating project directory... ✓
+Pulling down GitHub repository... ✓ nodejs
+
+Successfully created project my-starter-project.
+
+Your project is ready to be deployed by Architect!
+To deploy locally, run:
+  architect dev my-starter-project/architect.yml
 ```
 
-## Running Locally
-The `architect.yml` file is declarative, which allows the Architect Component it describes to be run in any environment,
-from local development all the way to production. Follow these steps to clone this repository and run the application
-locally.
+```sh
+$ cd ./my-starter-project
+```
 
-Once the deployment has completed, you can reach your new service by going to https://api.localhost.architect.sh.
+### Clone it yourself
+Run the following command to clone the repo yourself:
 
 ```sh
 # Clone the repository and navigate to this directory
 $ git clone git@github.com:architect-templates/nodejs.git
 $ cd ./nodejs
+```
 
-# Install node packages
-$ npm i
+## Run Locally
+Once the repo has been cloned to your local machine, execute the following command from the `nodejs` directory to run it locally:
 
-# Deploy locally using the dev command
-$ architect dev architect.yml
+```sh
+$ architect dev .
 ```
 
-## Deploying to the Cloud
+When this command completes, you can reach your new application at https://app.localhost.architect.sh.
+### Make your own changes
+
+This application's `architect.yml` file contains a `debug` block that enables hot-reloading for each service
+within the component. That means you can make changes to the source code, which will automatically apply to the environment. This lets you quickly iterate and see your changes without restarting the
+application stack.
+
+Give it a try! Search inside your project for “Favorite Movie” and change this string to "Favorite Pizza" Once you save
+the file, you’ll see the frontend service recompiling in the logs, and then your browser window will update automatically.
```

Comment:
The browser window doesn't update automatically when changes are made here, so this should be updated